src/components/Form.js

```diff
@@ -94,19 +130,25 @@ export default function Form({
       <br />
       <label htmlFor="paymentMethod">Payment Method</label>
       <br />
-      <select id="paymentMethod" name="paymentMethod" defaultValue={initialData.paymentMethod} required>
+      <select
+        id="paymentMethod"
+        name="paymentMethod"
+        defaultValue={initialData.paymentMethod}
+        required
+      >
         <option value="">Select payment method</option>
         <option value="Cash">Cash</option>
         <option value="Debit card">Debit card</option>
         <option value="Credit card">Credit card</option>
         <option value="PayPal">PayPal</option>
         <option value="Apple Pay">Apple Pay</option>
       </select>
-      <br />
-      <label htmlFor="description">Description</label>
-      <br />
-      <textarea id="description" name="description" rows="5" cols="50" placeholder="Add a description (optional)"
-                defaultValue={initialData.description} />
+      <br />{" "}
+      <Input
+        label="Description:"
+        name="description"
+        placeholder="Enter your description here..."
+      />
```

Comment:
According to your AC's the description field should be filled with the transactions description in edit mode. This is not the case, the description field is empty. Declare a defaultValue for the description field.